arroyo-state/src/committing_state.rs

```diff
@@ -1,10 +1,7 @@
-use crate::queries::controller_queries;
-use deadpool_postgres::Pool;
 use std::collections::HashSet;
-use std::time::SystemTime;
 
 pub struct CommittingState {
-    checkpoint_id: i64,
+    pub checkpoint_id: i64,
```

Comment:
rather than making these pub, can you just make an accessor?

Reply:
sure, done